db/schema.rb

```diff
@@ -52,7 +52,7 @@
   create_table "trips", force: :cascade do |t|
     t.bigint "origin_id", null: false
     t.bigint "destination_id", null: false
-    t.bigint "user_id", null: false
+    t.bigint "user_id"
```

Comment:
Why are we allowing trips to not have users?

Reply:
We only need a registred user to bookmark or add notifications.
Every other action is open

Reply:
see above